packages/pathfinder/src/pathfinder.ts

```diff
@@ -248,8 +329,29 @@ export class Pathfinder {
 
         // todo implement the pathfinding logic
         // be sure to use REFERENCE_ACCOUNT_PUBLIC_KEY as the destination and source accounts
-
-        return Promise.resolve([]);
+        return server.paths(Pathfinder.REFERENCE_ACCOUNT_PUBLIC_KEY, 
+                            Pathfinder.REFERENCE_ACCOUNT_PUBLIC_KEY, 
+                            destinationAsset, destinationAmount)
```

Comment:
Can't just use the destinationAsset here. Its just a POJO right now.

Reply:
fixed